packages/connect/src/api/firmwareUpdate.ts

```diff
@@ -34,15 +35,14 @@ export default class FirmwareUpdate extends AbstractMethod<'firmwareUpdate', Par
 
         const { payload } = this;
 
-        validateParams(payload, [
-            { name: 'version', type: 'array' },
```

Comment:
previously, it did not allow empty array. now it does, right? this isn't a big deal I'd say

Reply:
Oh, that's true. There is probably more cases like this, I will go through them